Core/include/Acts/Material/Material.hpp

```diff
@@ -35,7 +35,7 @@ class Material {
     eX0 = 0,
     eL0 = 1,
     eAr = 2,
-    eZ = 3,
+    eNuclearCharge = 3,
```

Comment:
I'm not sure I understand why you change this enum value and not the others?

Reply:
Because `Material::eZ` clashes with `AxisDefs::eZ`. I think, we should also rename the other enum values according to their meaning instead of their typical symbols, but here I only wanted to fix the compiler error.

Reply:
Right. Ok. Eventually, maybe `enum class` would be the way to go to avoid such clashes.

Reply:
In principle yes, but here the enum is used as an array index and must be implicitely convertible.

Reply:
Yeah sure, but if we go through a getter or similar, you can have a strong type, `static_cast` to the underlying int, and to the actual lookup. Gives proper type safety, and is guaranteed to be clash free.

Reply:
Sure, but that approach requires quite a bit more work.

Reply:
Ah yeah, totally.